`conform.nvim` support

Trying to integrate [`conform.nvim`](https://github.com/stevearc/conform.nvim) with `jqfmt`:

```lua
      formatters = {
        jqfmt = {
          command = os.getenv("XDG_DATA_HOME") .. "/go/bin/jqfmt",
          args = { "-ob", "-ar", "-op", "pipe" },
        },
    }
```

This config will fail with:
```
21:14:51[ERROR] Formatter 'jqfmt' error: time="2024-07-06T21:14:51+03:00" level=fatal msg="could not read file: open /dev/stdin: no such device or address"
```

FR opened: https://github.com/stevearc/conform.nvim/issues/396

`conform.nvim` will by default load the buffer into the formatter's stdin and read its stdout (`formatters.jqfmt.stdin = true`).

Any idea why and where `/dev/stdin` is gone and/or how to debug this?

--- a/cmd/jqfmt/main.go
+++ b/cmd/jqfmt/main.go
@@ -3,12 +3,12 @@ package main
 import (
 	"flag"
 	"fmt"
-	"io/ioutil"
+	"io"
+	"os"
 	"strings"
 
-	log "github.com/sirupsen/logrus"
-
 	"github.com/noperator/jqfmt"
+	log "github.com/sirupsen/logrus"
 )
 
 func assertErrorToNilf(message string, err error) {
@@ -18,7 +18,6 @@ func assertErrorToNilf(message string, err error) {
 }
 
 func main() {
-
 	// funcsStr := flag.String("fn", "", "functions")
 	opsStr := flag.String("op", "", "operators")
 	obj := flag.Bool("ob", false, "objects")
@@ -28,13 +27,14 @@ func main() {
 	verbose := flag.Bool("v", false, "verbose")
 	// noHang := flag.Bool("nh", false, "no hanging indent")
 	flag.Parse()
+	var from_stdin bool = false
 
 	if *verbose {
 		log.SetLevel(log.DebugLevel)
 	}
 
 	if *file == "" {
-		*file = "/dev/stdin"
+		from_stdin = true
 	}
 
 	// var funcs []string
@@ -61,13 +61,15 @@ func main() {
 	})
 	assertErrorToNilf("invalid config: %v", err)
 
-	// Read in program.
-	jqBytes, err := ioutil.ReadFile(*file)
+	var jqBytes []byte
+	if from_stdin {
+		jqBytes, err = io.ReadAll(os.Stdin)
+	} else {
+		jqBytes, err = os.ReadFile(*file)
+	}
 	assertErrorToNilf("could not read file: %v", err)
 	jqStr := string(jqBytes)
-
 	jqStrFmt, err := jqfmt.DoThing(jqStr, cliCfg)
 	assertErrorToNilf("could not format jq: %v", err)
-
 	fmt.Print(jqStrFmt)
 }

